modules/littlefs/CMakeLists.txt

```diff
@@ -0,0 +1,18 @@
+if(CONFIG_FILE_SYSTEM_LITTLEFS)
+  add_library(LITTLEFS INTERFACE)
+
+  target_include_directories(LITTLEFS INTERFACE
+    ${ZEPHYR_LITTLEFS_MODULE_DIR}
+    ${CMAKE_CURRENT_SOURCE_DIR}
+  )
+  target_link_libraries(LITTLEFS INTERFACE zephyr_interface)
```

Comment:
Why is this necessary?

Someone that links with LITTLEFS is not already linking with zephyr_interface ?

Which library is that ?

Reply:
note sure, code is moved from here: https://github.com/zephyrproject-rtos/littlefs/blob/zephyr/CMakeLists.txt

and I do not know if there is a reason, so I prefer not to change.

Changing this is outside the scope of moving gluecode into Zephyr and should be cleaned up independently.